frontend/src/components/Item/Card/Card.vue

```diff
@@ -83,176 +85,134 @@ import {
-      }
-    },
-    text: {
-      type: Boolean,
-      default: (): boolean => {
-        return false;
-      }
-    },
-    margin: {
-      type: Boolean,
-      default: (): boolean => {
-        return false;
-      }
-    },
-    link: {
-      type: Boolean,
-      default: (): boolean => {
-        return false;
-      }
-    }
-  },
-  setup() {
-    const taskManager = taskManagerStore();
+const props = withDefaults(
+  defineProps<{
+    item: BaseItemDto;
+    shape: string | boolean;
+    episode: boolean;
+    overlay: boolean;
+    text: boolean;
+    margin: false;
+    link: false;
+  }>(),
+  {
+    shape: false,
+    episode: false,
+    overlay: false,
+    text: false,
+    margin: false,
+    link: false
+  }
+);
 
-    return { taskManager };
-  },
-  computed: {
-    cardType(): string {
-      // Otherwise, figure out the shape based on the type of the item
-      return getShapeFromItemType(this.item.Type);
-    },
-    /**
-     * Either the item name or the series name
-     */
-    cardTitle(): string {
-      return this.item.Type !== 'Episode'
-        ? this.item.Name || ''
-        : this.item.SeriesName || '';
-    },
-    /**
-     * Returns either a string representing the production year(s) for the current item
-     * or the episode name of an item (SX EY - Episode Name)
-     * or the album artist
-     */
-    cardSubtitle(): string {
-      switch (this.item.Type) {
-        case 'Episode': {
-          return `${this.$t('seasonEpisodeAbbrev', {
-            seasonNumber: this.item.ParentIndexNumber,
-            episodeNumber: this.item.IndexNumber
-          })} - ${this.item.Name}`;
-        }
-        case 'MusicAlbum': {
-          return `${this.item.AlbumArtist || ''}`;
-        }
-        case 'Series': {
-          if (this.item.Status === 'Continuing') {
-            return `${this.item.ProductionYear} - ${this.$t('present')}`;
-          } else if (this.item.EndDate) {
-            const endYear = new Date(this.item?.EndDate).toLocaleString(
-              'en-us',
-              { year: 'numeric' }
-            );
+const taskManager = taskManagerStore();
+const { t } = useI18n();
 
-            if (this.item.ProductionYear?.toString() === endYear) {
-              return this.item.ProductionYear.toString();
-            }
+const cardType = computed(() => getShapeFromItemType(props.item.Type));
 
-            return `${this.item.ProductionYear} - ${endYear}`;
-          }
+const cardTitle = computed(() =>
+  props.item.Type !== 'Episode'
+    ? props.item.Name || ''
+    : props.item.SeriesName || ''
+);
 
-          break;
-        }
-        default: {
-          return `${this.item.ProductionYear || ''}`;
-        }
-      }
+/**
+ * Returns either a string representing the production year(s) for the current item
+ * or the episode name of an item (SX EY - Episode Name)
+ * or the album artist
+ */
+const cardSubtitle = computed(() => {
+  switch (props.item.Type) {
+    case 'Episode': {
```

Comment:
Not sure if applicable here, but don't we have a ``BaseItemKind`` enum exposed in the API? Should be available now that we're using unstable.

Revisit all the uses of item types in this file if that enum exists please (I also see it in the computed property above this statement for example).

Reply:
Done